packages/ckeditor5-dev-ci/vitest.config.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @license Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig( {
+	test: {
```

Comment:
For now this is fine, but if we are going to rewrite other packages, we need to store the config in a common place.